frontend/src/pages/EventCreationPage/components/EventCreationForm/index.tsx

```diff
@@ -29,9 +29,29 @@ interface InitialValuesType {
   hosts: OfficerNameData[];
 }
 
+function findNearestHour(date, isStartDate) {
+  let hour = date.getHours();
+  let startMinutes = date.getMinutes();
+  if (startMinutes === 0) {
+    startMinutes = '00';
```

Comment:
I think you should use a Number here instead of a string. I think for Date.setMinutes() and date-fns's setMinutes() they both have the minute param as type Number

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setMinutes
https://date-fns.org/v2.21.1/docs/setMinutes